ChoukaBaraGame/Scripts/Player/Player.gd

```diff
@@ -15,6 +15,7 @@ onready var homeBase = $HomeBase
 var pawnScene = preload("res://Scenes/Player/Pawn.tscn")
 var maxCharactersPerPlayer = 4
 var needsReRoll:bool = false
+var canEnterInnerCircle = false
```

Comment:
What is this used for?

Reply:
Pawn can enter inner circle only when the player has a hit under his belt. other wise he has to move another pawn.
This is the flag at player level to track that.